s.el

```diff
@@ -233,9 +233,9 @@ When not specified, ELLIPSIS defaults to ‘...’."
   (declare (pure t) (side-effect-free t))
   (let ((extra (max 0 (- len (length s)))))
     (concat
-     (make-string (ceiling extra 2) ? )
+     (make-string (ceiling extra 2) ?)
```

Comment:
`(? )` is an unfortunate way of writing a space character (inside a list).
`(?)` is a syntax error.

I recommend you use `(?\s)`

Reply:
Nice catch.